application/src/main/java/org/thingsboard/server/service/transport/DefaultTransportApiService.java

```diff
@@ -664,4 +715,13 @@ private ListenableFuture<TransportApiResponseMsg> queuesToTransportApiResponseMs
     private Long checkLong(Long l) {
         return l != null ? l : 0;
     }
+
+    private String extractRegex(String commonName, String regex) {
```

Comment:
I think method name could be improved to **extractDeviceNameFromCNByRegEx** or similar.